include/oneapi/dpl/pstl/glue_memory_impl.h

```diff
@@ -110,13 +107,13 @@ uninitialized_move(_ExecutionPolicy&& __exec, _InputIterator __first, _InputIter
     {
         return oneapi::dpl::__internal::__pattern_walk2_brick(
             __dispatch_tag, std::forward<_ExecutionPolicy>(__exec), __first, __last, __result,
-            oneapi::dpl::__internal::__brick_copy<decltype(__dispatch_tag), _DecayedExecutionPolicy>{});
+            oneapi::dpl::__internal::__brick_copy<decltype(__dispatch_tag)>{});
     }
     else
     {
         return oneapi::dpl::__internal::__pattern_walk2(
             __dispatch_tag, std::forward<_ExecutionPolicy>(__exec), __first, __last, __result,
-            oneapi::dpl::__internal::__op_uninitialized_move<_DecayedExecutionPolicy>{});
+            oneapi::dpl::__internal::__op_uninitialized_move<std::decay_t<_ExecutionPolicy>>{});
```

Comment:
As far as we are changing template parameters of `__op_uninitialized_move`....
It looks `__op_uninitialized_move` has only one implementation for the host and device purposes. So, let remove typename... _ExecutionPolicy at all.

Reply:
Fixed

Reply:
I don't know from implementation point of view how to do this.
@MikeDvorskiy could you please prepare some separate PR with this fix?